samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/pipeline.py

```diff
@@ -0,0 +1,62 @@
+import kfp.dsl as dsl
+from kfp.gcp import use_gcp_secret
+
+@dsl.pipeline(
+    name = "kaggle pipeline",
+    description = "kaggle pipeline that go from download data, train model to display result"
+)
+def kaggle_houseprice(
+    gcr_address: str,
+    bucket_name: str
+):
+    import os
+    stepDownloadData = dsl.ContainerOp(
```

Comment:
Same here. Let's componentize all these ops.
Each component will go into its own `component.yaml` file and the build script will replace the image versions inside those files, not in the `pipeline.py`.

Reply:
Thanks. Will do.